components/asp/database/tables.md

```
@@ -0,0 +1,49 @@
+| fileTree           |  STRING     | JSON String of the filetree for the program     |
+
+
+### Traces
+
+Name: <sys_id>_<sys_ver>_traces
+
+| Column Name        |  Data Type  |  Description |
+|--------------------|:-----------:|------------------------------------------------:|
+| deploymen_id       |  INTEGER    | Deployment Id (eg 554233)                       |
+| trace_id           |  INTEGER    | Trace Id (eg 235234)                            |
+| trace_id           | INTEGER    | Trace Id (eg 235234)                            
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)